### PR TITLE
Update templates for new lifecycle

### PR DESCRIPTION
**Pull request by adrianstevens**

### CODE REVIEW

File: Meadow.CSharp.Template/MeadowApp.cs [hunk 1]
```diff
@@ -59,10 +65,9 @@ void ShowColorPulse(Color color, TimeSpan duration)
 
 		void ShowColor(Color color, TimeSpan duration)
 		{
-			Console.WriteLine($"Color: {color}");
```
Comment (patridge) [inline Meadow.CSharp.Template/MeadowApp.cs:62]: The rest of this class looks like lifecycle updates. Not opposed here, but was this removal intentional? It is still present in the F# and VB versions.

File: Meadow.FSharp.Template/Program.fs [hunk 2]
```diff
@@ -1,40 +1,55 @@
...
-    let led = new RgbPwmLed(MeadowApp.Device, MeadowApp.Device.Pins.OnboardLedRed,MeadowApp.Device.Pins.OnboardLedGreen, MeadowApp.Device.Pins.OnboardLedBlue,Meadow.Peripherals.Leds.IRgbLed.CommonType.CommonAnode)
-    
-    let ShowColorPulse (color : Color) (duration : TimeSpan)  = 
+    // Change F7MicroV2 to F7Micro for V1.x boards
+    inherit App<F7FeatherV2>()
+
+    let mutable led : RgbPwmLed = 
+        null
+
+    let ShowcolorPulse (color : Color) (duration : TimeSpan) =
```
Comment (patridge) [inline Meadow.FSharp.Template/Program.fs:17]: I'm not a great candidate for F# review, but this casing change didn't seem intentional.

```suggestion
    let ShowColorPulse (color : Color) (duration : TimeSpan) = 
```

File: Meadow.FSharp.Template/Program.fs [hunk 3]
```diff
@@ -1,40 +1,55 @@
...
-            
-    do cyclecolors (TimeSpan.FromSeconds(1))
-
-[<EntryPoint>]
-let main argv =
-    let app = new MeadowApp()
-    Threading.Thread.Sleep (System.Threading.Timeout.Infinite)
-    0 // return an integer exit code
\ No newline at end of file
+            ShowcolorPulse Color.Blue duration
```
Comment (patridge) [inline Meadow.FSharp.Template/Program.fs:26]: Ditto for all of these `ShowcolorPulse` calls (vs. `ShowColorPulse` with a capital C).

**Review by patridge — APPROVED** [review]
LGTM, though had a few questions.

File: Meadow.CSharp.Template/MeadowApp.cs (hunk 1 above)
Comment (adrianstevens) [reply Meadow.CSharp.Template/MeadowApp.cs:62]: yeah did a little cleanup and consistency while I was in there

File: Meadow.FSharp.Template/Program.fs (hunk 2 above)
Comment (adrianstevens) [reply Meadow.FSharp.Template/Program.fs:17]: It was - the C should be uppercase

File: Meadow.CSharp.Template/MeadowApp.cs (hunk 1 above)
Comment (patridge) [reply Meadow.CSharp.Template/MeadowApp.cs:62]: Okay...I see it now. The other cycle colors logging is there, but this method isn't used at all here. Should we just delete it all?

**Review by CartBlanche — APPROVED** [review]
LGTM
